src/loader/objects/enemy.cpp

```diff
@@ -6,14 +6,24 @@ namespace loader {
     void Enemy::load(FILE *file) {
         LoadObject::resetTransferred();
 
-        int numeroNemici;
-        fscanf(file, "%d", &numeroNemici);
-        this->loadedObjects_->resize(numeroNemici);
-        for (int i = 0; i < numeroNemici; i++) {
+        int numeroShooter;
+        fscanf(file, "%d", &numeroShooter);
+        this->loadedObjects_->resize(numeroShooter);
+        for (int i = 0; i < numeroShooter; i++) {
             Position startPosition;
             fscanf(file, "%d %d\n", &startPosition.riga, &startPosition.colonna);
-            entities::Enemy *enemy = new entities::Enemy(startPosition);
+            entities::Enemy *enemy = new entities::Shooter(startPosition);
             this->loadedObjects_->at(i) = enemy;
         }
+        
+        //int numeroKamikaze;
+        //fscanf(file, "%d", &numeroKamikaze);
+        //this->loadedObjects_->resize(numeroKamikaze);
+        //for (int i = 0; i < numeroKamikaze; i++) {
+        //    Position startPosition;
+        //    fscanf(file, "%d %d\n", &startPosition.riga, &startPosition.colonna);
+        //    entities::Enemy *enemy = new entities::Kamikaze(startPosition);
+        //    this->loadedObjects_->at(i) = enemy;
+        //}
```

Comment:
Questa parte è ancora TODO vero?

Reply:
è già fatta, vanno cambiate le mappe in modo da usare una volta un nemico e una volta un'altro